atomate/vasp/fireworks/tests/test_lobster.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+
+
+import os
+import unittest
+
+from atomate.vasp.fireworks.core import StaticFW
+from atomate.vasp.fireworks.lobster import LobsterFW
+from pymatgen.core.lattice import Lattice
+from pymatgen.core.structure import Structure
+
+__author__ = 'Janine George, Guido Petretto'
+__email__ = 'janine.george@uclouvain.be'
+
+module_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)))
+db_dir = os.path.join(module_dir, "..", "..", "..", "common", "test_files")
+reference_dir = os.path.join(module_dir, "..", "..", "test_files")
+
+DEBUG_MODE = False  # If true, retains the database and output dirs at the end of the test
+VASP_CMD = None  # If None, runs a "fake" VASP. Otherwise, runs VASP with this command...

```

Comment:
You can delete these? They aren't doing anything currently.

Reply:
yes